app/views/admin/cx_collections/index.html.erb

```diff
@@ -60,10 +60,10 @@
     <tr>
       <td class="text-base text-sm">
         for
-        <%- if @quarter != 0 %>
+        <%- if @quarter != 0 && @quarter != 'All' %>
         Q<%= params["quarter"] %>
         <% end %>
-        <%- if @year != 0 %>
+        <%- if @year != 0 && @year != 'All' %>
         <%= params["year"] %>
         <% end %>
       </td>
```

Comment:
When Year = All or Quarter = All, the second row of the table reports all zeros. Should that be the case?

<img width="1390" alt="Screenshot 2024-09-24 at 9 34 12 AM" src="https://github.com/user-attachments/assets/67ac6f55-ad6f-474a-8257-d7f2d6248413">

<img width="1391" alt="Screenshot 2024-09-24 at 9 37 22 AM" src="https://github.com/user-attachments/assets/0ac3b3a8-0aaa-4f14-96ed-a4301bd99f6d">